techeria_controller/techeria_app/models.py

```diff
@@ -4,13 +4,13 @@
 # Create your models here.
 
 class Products(models.Model):
+    image = models.ImageField(null=False, blank=False, upload_to='images/')
     name = models.CharField(max_length=255)
```

Comment:
Also how about referencing the categories  with a URL  using slug. ex: slug = models.SlugField(max_length = 250, null = True, blank = True)